about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="Description" content="first capstone project" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>AYSD</title>
+</head>
+
+<body>
+    <header>
+        <img src="./imgs/website/landingImg.png" alt="background image" class="bg-img">
+        <div class=" d-none d-md-flex banner  align-items-center">
+            <div class="container d-md-flex justify-content-end px-md-5">
+                <div class="icons">
+                    <a href="#" rel="noopener" class="px-2"><i class="fab fa-facebook-f"></i></a>
+                    <a href="#" rel="noopener" class="px-2"><i class="fab fa-twitter"></i></a>
+                    <a href="#" rel="noopener" class="px-2">English</a>
+                    <a href="#" rel="noopener" class="px-2">Logout</a>
+                </div>
+            </div>
+        </div>
+        <nav class="navbar navbar-expand-md ">
+            <div class="container px-md-5">
+                <a class="navbar-brand d-none d-md-flex" href="index.html"><img class="w-25 logoImg"
+                        src="./imgs/website/شعار_شبكة_الشباب_العربي_للتنميةالمستدامة_المعتمد-removebg-preview.png"
+                        alt="the logo of the org"></a>
+                <button class="navbar-toggler" type="button">
+                    <span class="navbar-toggler-icon text-third"></span>
+                </button>
+                <div class=" flex-grow-0 d-none d-md-flex" id="navbarSupportedContent">
+                    <a href="javascript:void(0)" class="closebtn d-md-none">&times;</a>
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item ms-md-3 ms-lg-5">
+                            <a class="nav-link active" aria-current="page" href="about.html">About</a>
```

Comment:
- [ ] You did great work @Mofuhidy 😃, however, in this `about.html` page we are not supposed to have again the `about` button in your navbar. What do you think if we change it to this `<a class="nav-link active" aria-current="page" href="index.html">Home</a>` to facilitate the user to go back to the `home` page 👍. Kindly fix it and make sure it is the same for the `mobile menu` 💪.